docs/examples.rst

```diff
@@ -634,20 +634,32 @@ package then could be done in the following way::
 Parametrize Plans
 ------------------------------------------------------------------
 
-It is possible to parametrize plans using environment variables.
-This may be useful to reduce duplication, for example in CI systems.
-The syntax is standard, both ``$var`` and ``${var}`` may be used.
-The values of variables are taken from the ``--environment``
-command line option and the ``environment`` plan attribute.
-If a variable is defined using both the attribute and the option,
-the value from the ``--environment`` option has priority::
+It is possible to parametrize plans using environment variables and
+context. This may be useful to reduce duplication, for example in
+CI systems.
+
+For environment variables the syntax is standard, both ``$var`` and
+``${var}`` may be used. The values of variables are taken from the
+``--environment`` command line option and the ``environment`` plan
+attribute. If a variable is defined using both the attribute and
+the option, the value from the ``--environment`` option has
+a priority::
 
     discover:
         how: fmf
         url: https://github.com/teemtee/${REPO}
 
     $ tmt run -e REPO=tmt
 
+For context parametrization the syntax is ``$@dimension`` or
+``$@{dimension}``. The values are set according to the defined
+context specified using ``--context`` command line option and the
+``context`` plan attribute::
+
+    context:
+        repo: tmt
```

Comment:
Would be nice to have example where `$@{repo}` is used.

Reply:
Updated